src/Makefile.am

```diff
@@ -436,8 +438,8 @@ nodist_libbitcoin_util_a_SOURCES = $(srcdir)/obj/build.h
 #
 
 # bitcoind binary #
-bitcoind_SOURCES = bitcoind.cpp
-bitcoind_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+bitcoind_SOURCES = bitcoind.cpp tinyformat/tinyformat.h
```

Comment:
I'm unclear on why this is required to be added to sources, but the build failed consistently when it was only provided via `-I` below, e.g.:
https://travis-ci.org/Empact/bitcoin/jobs/410971896